### PR TITLE
feat: Tutorial 12.6 CSS Grid

### PR DESCRIPTION
**Pull request by Anisha-Alexander**

### CODE REVIEW

File: Tutorial12.6/12-column-gridsys.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            display: grid;
+
+        }
+    </style>
+</head>
+<body>
+    <input type="checkbox"/>
+    <div id="content">
+        <div id="grid">
+            <p></p><p></p><p></p><p></p><p></p><p></p><p></p><p></p><p></p><p></p><p></p><p></p>
```
Comment (donjude-eng) [inline Tutorial12.6/12-column-gridsys.html:75]: ?

Comment (donjude-eng) [reply Tutorial12.6/12-column-gridsys.html:75]: P tags

File: Tutorial12.6/responsive-layout.html [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            "header header header header"
+            "footer footer footer footer"
+            "main main main main"
+            "main main main main"
+            "aside aside nav nav"
+            "section section section section"
+            "section section section section";
+        }
+    @media screen and (min-width:760px){ 
+        #content{
```
Comment (donjude-eng) [inline Tutorial12.6/responsive-layout.html:28]: 768px is the standard breakpoint for tab screens

**Review by donjude-eng — CHANGES_REQUESTED** [review]

File: Tutorial12.6/responsive-layout.html (hunk 2 above)
Comment (Anisha-Alexander) [reply Tutorial12.6/responsive-layout.html:28]: ok

**Pull request merged**